doc/news/changes/minor/20190807Sanchez-Vizuet

```diff
@@ -0,0 +1,4 @@
+Add references to the introduction of step 51
+Include the references in the bibliography
+
+(Tonatiuh Sanchez-Vizuet August 7, 2019)
```

Comment:
As Wolfgang said, this format is not quite correct. Use ``<br>`` and format the date exactly how the other files in the same folder are formatted.